kornia/color/hls.py

```diff
@@ -67,20 +58,20 @@ def rgb_to_hls(image: torch.Tensor, eps: float = 1e-8) -> torch.Tensor:
         torch.sub(maxc, minc, out=s)  # s = max - min
 
     # precompute image / (max - min)
-    im: torch.Tensor = image / (s + eps).unsqueeze(-3)
+    im = image / (s + eps).unsqueeze(-3)
 
     # epsilon cannot be inside the torch.where to avoid precision issues
-    s /= torch.where(l_ < 1.0, l_, 2.0 - l_) + eps  # saturation
+    s /= where(l_ < 1.0, l_, 2.0 - l_) + eps  # saturation
     l_ /= 2  # luminance
 
     # note that r,g and b were previously div by (max - min)
-    r: torch.Tensor = torch.select(im, -3, 0)
-    g: torch.Tensor = torch.select(im, -3, 1)
-    b: torch.Tensor = torch.select(im, -3, 2)
+    r = torch.select(im, -3, 0)
```

Comment:
```suggestion
    r = im[…, 0, :,:]
```
Use more pythonic expressions in rgb to hls

Reply:
Added as comment to open PR for further improvements